depthai_sdk/setup.py

```diff
@@ -1,44 +1,51 @@
 import io
+from setuptools import setup, find_packages
 
-from setuptools import setup
-
+# Read requirements from requirements.txt
 with open('requirements.txt') as f:
     requirements = f.readlines()
 
-install_requires = [requirement for requirement in requirements if '--' not in requirement]
+# Flatten the install_requires list
+install_requires = [requirement.strip() for requirement in requirements if '--' not in requirement]
 
 setup(
     name='depthai-sdk',
     version='1.15.0',
     description='This package provides an abstraction of the DepthAI API library.',
-    long_description=io.open("README.md", encoding="utf-8").read(),
+    long_description=open('README.md').read(),
     long_description_content_type="text/markdown",
-    url='https://github.com/luxonis/depthai/tree/main/depthai_sdk',
+    url='https://github.com/dbrown99c/depthai/tree/main/depthai_sdk',
```

Comment:
Can't approve that. Please fix this first before we can merge.

Reply:
removed all the extraneous commits apologies thank you